discovery/http/http.go

```diff
@@ -122,6 +123,7 @@ func NewDiscovery(conf *SDConfig, logger log.Logger) (*Discovery, error) {
 		url:             conf.URL,
 		client:          client,
 		refreshInterval: time.Duration(conf.RefreshInterval), // Stored to be sent as headers.
+		tgLastLength:     0,
```

Comment:
that is not needed as it is the default value.

Reply:
I deleted it.